src/django_components/template_loader.py

```diff
@@ -23,7 +23,27 @@ def path_is_relative_to(child_path: str, parent_path: str) -> bool:
 
 
 class Loader(FilesystemLoader):
+    # TODO - Look deeper into how this function works
+    #      -> Are STATICFILES_DIRS absolute paths or relative to project root? README suggest absolute,
+    #         but the default suggests relative, and code suggests both. Document all the supported cases.
+    #      -> Why do we add settings directory?
+    #      -> Could we merge this with `autodiscover_modules()` in `autodiscover.py`, since there we also
+    #         look for `[app_name]/components`, or do these two accomplish different things?
```

Comment:
Questions:

1. Are STATICFILES_DIRS absolute paths or relative to project root? README suggest absolute, but the default suggests relative, and code suggests both. Document all the supported cases.
	- I've already touched on this in other comments . But [Django asks for full paths](https://docs.djangoproject.com/en/5.0/ref/settings/#staticfiles-dirs).

2. Why do we add settings directory?
	- Addressed in https://github.com/EmilStenstrom/django-components/pull/533/files#r1649615698

3. Could we merge this with `autodiscover_modules()` in `autodiscover.py`, since there we also look for `[app_name]/components`, or do these two accomplish different things?
	- No. `autodiscover_modules` imports individual modules (AKA files), whereas in `STATICFILES_DIRS`, we set DIRS as inputs.

Reply:
TODO: Use this for updating README